controlplane/pkg/plugins/utils.go

```diff
@@ -97,3 +101,27 @@ func registerPlugin(name, endpoint, registry string, capabilities []plugins.Plug
 
 	return err
 }
+
+func monitorNSMD(name string, endpoint string, registry string, capabilities []plugins.PluginCapability) {
+	registered := true
+	for ; ; time.After(livenessCheckPeriod) {
+		resp, err := http.Get("http://localhost:5555/liveness")
```

Comment:
can we get rid of using global client?

Reply:
> can we get rid of using global client?

@denis-tingajkin could you please elaborate on this?

Reply:
@alex-yust http.Get() uses global client. We could 
initialize and use instance of htpp client before cycle.

Reply:
- [ ] use local http.Client